pylabrobot/liquid_handling/backends/tecan/EVO_backend.py

```diff
@@ -1100,15 +1224,291 @@ async def get_disposable_tip(self, tips, z_start, z_search):
       params=[tips, z_start, z_search, 0],
     )
 
-  async def discard_disposable_tip(self, tips):
+  async def discard_disposable_tip_high(self, tips):  # changed by VIKMOL
     """Drops tips
-
+    Discards at the Z-axes initialization hight
     Args:
       tips: binary coded tip select
     """
 
     await self.backend.send_command(module=self.module, command="ADT", params=[tips])
 
+  async def discard_disposable_tip(self, tips, discard_hight):  # added by VIKMOL z_start
+    """Drops tips
+    Discards at a veriable Z-axes initialization hight
+    Args:
+      tips: binary coded tip select
+      discard_hight: binary. 0 above tip rack, 1 in tip rack
+    """
+
+    await self.backend.send_command(module=self.module, command="AST", params=[tips, discard_hight])
+
+  async def place_tip_back(self, tips, discard_hight):  # added by VIKMOL z_start, z_search
+    """Places tips back into the original tip rack positions.
+
+    Args:
+      tips: Binary coded tip select
+      discard_hight: binary. 0 above tip rack, 1 in tip rack
+
+    """
+
+    await self.backend.send_command(module=self.module, command="AST", params=[tips, discard_hight])
```

Comment:
`return_tips` calls this without the `discard_hight`. perhaps `return_tips` is not used anymore?